project/deps.sc

```diff
@@ -31,9 +31,10 @@ implicit class DepOps(private val dep: Dep) {
 }
 
 object Deps {
-  def ammoniteCompiler(sv: String)             =
-    if (sv.startsWith("2.")) ivy"com.lihaoyi:::ammonite-compiler:${Versions.ammonite}"
-    else ivy"com.lihaoyi:ammonite-compiler_$sv:${Versions.ammonite}"
+  
+  def ammoniteCompiler(sv: String) = {
+      if (sv.startsWith("2.")) ivy"com.lihaoyi:::ammonite-compiler:${Versions.ammonite}"
+      else ivy"com.lihaoyi:ammonite-compiler_$sv:${Versions.ammonite}"}
```

Comment:
You probably want to revert the changes here :wink:

Reply:
I think that it was trying to move this artefact to the "native" scala 3 which caused the trouble. 

I suspect the 3 ammonite artefacts are quite closely coupled and I should not have attempted to tamper with them. 

I've compared these lines with the existing master branch, and as far as I can tell, they are the same... from my experiments, this PR now behaves as expected, with the confusing behaviour of reporting a 2.13 version string. 

Maybe this PR helps the maintainers, maybe not - at least the intent was the right one!  I think I'll down tools here though.

Thankyou again for time and suggestions, I learned a lot and it was appreciated.

Reply:
> I've compared these lines with the existing master branch, and as far as I can tell, they are the same

They're not. Just have a look here https://github.com/almond-sh/almond/pull/902/files
:wink: